contrib/cirrus/add_second_partition.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+# N/B: This script could mega f*!@up your disks if run by mistake.
+#      it is left without the execute-bit on purpose!
+
+# $SLASH_DEVICE is the disk device to be f*xtuP
+SLASH_DEVICE="/dev/sda"  # Always the case on GCP
+NEW_PART_START="50%"
+NEW_PART_END="100%"
+
+set -eo pipefail
+
+source $(dirname $0)/lib.sh
+
+if [[ ! -r "/root" ]] || [[ -r "/root/second_partition_ready" ]]
+then
+    echo "Warning: Ignoring attempted execution of $(basename $0)"
+    exit 0
+fi
+
+[[ -n "type -P parted" ]] || \
+    die 2 "The parted command is required."
+
+[[ ! -b ${SLASH_DEVICE}2 ]] || \
+    die 5 "Found unexpected block device ${SLASH_DEVICE}2"
+
+PPRINTCMD="parted --script ${SLASH_DEVICE} print"
+FINDMNTCMD="findmnt --source=${SLASH_DEVICE}1 --mountpoint=/ --canonicalize --evaluate --first-only --noheadings"
+TMPF=$(mktemp -p '' $(basename $0)_XXXX)
+trap "rm -f $TMPF" EXIT
+
+if $FINDMNTCMD | tee $TMPF | egrep -q "^/\s+${SLASH_DEVICE}1"
+then
+    echo "Repartitioning original partition table:"
+    $PPRINTCMD
+else
+    die 5 "Unexpected output from '$FINDMNTCMD': $TMPF"
```

Comment:
This just prints the name of the tempfile, not the findmnt output. Use `$(<$TMPF)` to output the contents.

Reply:
oh neat!  And good catch, thanks!

Reply:
fixed.